corehq/toggles.py

```diff
@@ -367,6 +367,14 @@ def toggle_values_by_name(username=None, domain=None):
     [NAMESPACE_DOMAIN, NAMESPACE_USER]
 )
 
+VELLUM_BETA = PredictablyRandomToggle(
+    'vellum_beta',
+    'Use Vellum beta version',
+    TAG_ONE_OFF,
+    [NAMESPACE_DOMAIN],
+    randomness=0.1
```

Comment:
Just because I just didn't have a method for selecting the "others," decided to pick them randomly. The predictably random toggle lets you also specify domains the should have the flag on, in addition to the random 10%, so yes, I'll add the domains with the case mgmt feature flags turned on to this new flag. Which would be messy to maintain, but I'm not expecting this code to live for long.

It's easy to tell if a specific domain has the flag turned on (by running `deterministic_random`), less easy to get a list of affected domains (would need to run all domains through `deterministic_random`...which, thinking about it, wouldn't be a bad addition to the feature flag page if it isn't already there).